src/conv-core/converse.h

```diff
@@ -1785,34 +1792,36 @@ typedef void (*CcdCondFn)(void *userParam);
 typedef void (*CcdVoidFn)(void *userParam,double curWallTime);
 
 /*CPU conditions*/
-#define CcdPROCESSOR_BEGIN_BUSY 0
-#define CcdPROCESSOR_END_IDLE 0 /*Synonym*/
-#define CcdPROCESSOR_BEGIN_IDLE 1
-#define CcdPROCESSOR_END_BUSY 1 /*Synonym*/
-#define CcdPROCESSOR_STILL_IDLE 2
-#define CcdPROCESSOR_LONG_IDLE 3
+#define CcdSCHEDLOOP            0
```

Comment:
Ideally this whitespace reformatting would be in a different commit, but it's unlikely to create problems here.